app/package.json

```diff
@@ -6,13 +6,13 @@
   "main": "./main.js",
```

Comment:
Version must be also updated here

Reply:
Done https://github.com/cerebroapp/cerebro/pull/594/commits/ed45724c03a0dc2e0b368f99761ceee5ce901d7e